package.json

```diff
@@ -51,8 +51,9 @@
     "build": "babel --ignore=__tests__ src --out-dir lib",
     "lint": "standard",
     "prepublish": "npm test",
-    "posttest": "NODE_ENV=production npm run build",
+    "pretest": "npm run build",
```

Comment:
It should catch a type error if there is one. We're compiling with runtime checks before the test, and recompiling without after the test.